content/en/quickstart/device-to-device.md

```diff
@@ -11,3 +11,243 @@ menu:
     weight: 20
 toc: true
 ---
+This guide will walk you through the process of discovering/on-boarding/controlling your secure [OCF Device](https://openconnectivity.org/specs/OCF_Device_Specification_v2.2.3.pdf) using **plgd/sdk** library on your PC.
+
+## Prerequisite
+- The Cloud Server example should be running on your Raspberry Pi at the step of [Create device](../create-device)
+
+## Install OCF Client example
+1. Checkout [plgd/sdk](https://github.com/plgd-dev/sdk/tree/v2) library:
+    ```shell script
+    # execute commands on your PC
+    git clone -b v2 https://github.com/plgd-dev/sdk.git --recursive
+    cd sdk
+    ```
+2. Build OCF Client example:
+    ```shell script
+    cd cmd/ocfclient
+    go build
+    ```   
+
+3. Run OCF Client example in the same folder:
+    ```shell script
+    ./ocfclient 
+    ```
+
+## Discover device on local network 
```

Comment:
I like that you created a console application, but I would like to see here code snippets. Initialize Client - short code snippet including initialization and link to ocf client where is a full example.

Discover devices on local network 
code snippet to discover and print devices + link to example.

And at the end, usage of example client - and add there what you did :) Don't remove it. Just add here also the same using code.